crates/hotshot/src/tasks/mod.rs

```diff
@@ -58,162 +47,94 @@ pub enum GlobalEvent {
 /// # Panics
```

Comment:
I think these `# Panics` sections could be removed for almost all of these `add_*_task` functions now, since they no longer need an `unwrap` on the task id (except `add_consensus_task`)

Reply:
Still need it for creating the consensus state, but got rid of the rest